src/common/preload-temp-home.c

```diff
@@ -18,7 +18,6 @@
  */
 
 /* for RTLD_NEXT */
```

Comment:
This comment should be dropped along with the define. I'll wait for the current test run and re-push.